tests/main/proxy/task.yaml

```diff
@@ -1,18 +1,11 @@
 summary: Ensure that the core.proxy.* settings are honored
 
-# Limited to ubuntu 18.04 because the squid there is well tested. We
-# could add a squid (or custom go proxy) snap instead and run
-# everywhere.
-systems: [ubuntu-18.04-64]
-
 restore: |
     snap set core proxy.https=
-    apt autoremove --purge -y squid
+    systemctl stop tinyproxy
 
 prepare: |
-    # ensure we start from a clean state
-    rm -rf /var/log/squid/*
-    apt install -y squid
+    systemd-run --unit tinyproxy -- python3 $(pwd)/tinyproxy.py
```

Comment:
I think it should be "$(pwd)/tinyproxy.py"

Reply:
I haven't tried it with python 3 :-)

Reply:
ah, but it looks like mvo ported it. Hah! good'un